README.md

```diff
@@ -19,14 +19,27 @@ Libraries that produce telemetry data should only depend on `api`
 and defer the choice of the SDK to the application developer. Applications may
 depend on `sdk` or another package that implements the API.
 
-To install the API and SDK packages,
+All packages are published to [https://pkg.go.dev/go.opentelemetry.io/otel](https://pkg.go.dev/go.opentelemetry.io/otel) and is the preferred location to import from.
```

Comment:
```suggestion
All packages are published to [go.opentelemetry.io/otel](https://pkg.go.dev/go.opentelemetry.io/otel) and is the preferred location to import from.
```

My suggestion would also be to add links to https://blog.golang.org/using-go-modules and https://golang.org/cmd/go/#hdr-Add_dependencies_to_current_module_and_install_them as pointers about adding a dependency to the project.